src/app/backend/handler/filter.go

```diff
@@ -76,6 +83,19 @@ func formatRequestLog(request *restful.Request) string {
 		}
 	}
 
+	// Is DEBUG level logging enabled?
+	if args.Holder.GetAPILogLevel() != "DEBUG" {
+		// Great now let's filter out any content from sensitive URLs
```

Comment:
I would suggest is to export it to another method. You can also use map instead of slice to get rid of the loop:

```
_, isSensitive := sensitiveUrls[key]
```

You can put `struct{}` as a value to save space.

Please remember to use camelCase and switch likes of `sensitive_urls` to `sensitiveUrls`.

Reply:
Addressed and pushed to c820847a9468e26fb5f3baf06950a11da210989a -- Thoughts?

If that looks good, I can squash before the merge.